seerep-hdf5/seerep-hdf5-fb/src/hdf5-fb-general.cpp

```diff
@@ -89,16 +86,16 @@ void Hdf5FbGeneral::writeBoundingBoxLabeled(
 
 void Hdf5FbGeneral::writeBoundingBox2DLabeled(
     const std::string& datatypeGroup, const std::string& uuid,
-    const flatbuffers::Vector<flatbuffers::Offset<seerep::fb::BoundingBox2DLabeled>>* boundingbox2DLabeled)
+    const flatbuffers::Vector<flatbuffers::Offset<seerep::fb::BoundingBox2DLabeled>>& boundingbox2DLabeled)
 {
   std::string id = datatypeGroup + "/" + uuid;
 
-  if (boundingbox2DLabeled && !boundingbox2DLabeled->size() == 0)
+  if (!boundingbox2DLabeled.size() == 0)
```

Comment:
same for this: are you sure that this doesn't create a segmentation fault if the boundingbox2DLabeled vector in the FB msg isn't set?